unstable/wlr-screencopy-unstable-v1.xml

```diff
@@ -203,5 +203,18 @@
       <arg name="width" type="uint" summary="current width"/>
       <arg name="height" type="uint" summary="current height"/>
     </event>
+
+    <!-- Version 3 additions -->
+    <event name="linux_dmabuf" since="3">
+      <description summary="linux-dmabuf buffer information">
+        Provides information about the frame's linux-dmabuf buffer.
+
+        This event is sent once before the regular "buffer" event if
```

Comment:
Maybe we should add a `buffer_done` event instead relying on `buffer`?

Reply:
Yes, that would make it possible for compositors which implement the protocol to omit `wl_shm`.